src/data.js

```diff
@@ -2064,6 +2064,28 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Anatoliy Kirsantov',
+    description:
+      'Frontend Developer. UI/UX designer. Mobile. Startups Founder.',
+    url: 'https://kirsantov.com/uses/',
```

Comment:
I'm having trouble accessing your site (and I think everyone will)

See https://downforeveryoneorjustme.com/kirsantov.com

Let me know when this is fixed 😄